pkg/osquery/table/mdm.go

```diff
@@ -12,6 +12,40 @@ import (
 	"github.com/pkg/errors"
 )
 
+type profilesOutput struct {
+	ComputerLevel []profilePayload `plist:"_computerlevel"`
+}
+
+type profilePayload struct {
+	ProfileIdentifier  string
+	ProfileInstallDate string
+	ProfileItems       []profileItem
+}
+
+type profileItem struct {
+	PayloadContent *payloadContent
+	PayloadType    string
+}
+
+type payloadContent struct {
+	AccessRights            int
+	CheckInURL              string
+	ServerURL               string
+	ServerCapabilities      []string
+	Topic                   string
+	IdentityCertificateUUID string
+	SignMessage             bool
+}
+
+type profileStatus struct {
+	DEPEnrolled  bool
+	UserApproved bool
+}
+
+type depStatus struct {
```

Comment:
does this struct serve any purpose? Why doesn't `depStatus` return a simple boolean?